<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogColors</class>
 <widget class="QDialog" name="DialogColors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Palette editor</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,0">
    <item row="1" column="1">
     <widget class="QComboBox" name="cbmColors"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnColor">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbmBitplanes">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>no bitplanes</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Current color</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>RGB:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>RGB 4 bit (Amiga):</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="leRGB"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="leRGB4"/>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cbmPalette">
      <item>
       <property name="text">
        <string>Greyscale</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TwoColors</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btnLoad_2">
      <property name="text">
       <string>Set palette</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnLoad">
      <property name="text">
       <string>Load palette</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>Save palette</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave_2">
      <property name="text">
       <string>Export amiga palette</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblRGB4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
